### PR TITLE
OSDOCS#8482: updating RHEL update version numbers

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-8482](https://issues.redhat.com/browse/OSDOCS-8482)

Version(s): 4.15+

QE review:
- [x] QE has approved this change.

Preview: [Updating RHEL compute machines in your cluster](https://70443--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-cluster-rhel-compute#rhel-compute-updating-minor_updating-cluster-rhel-compute)

### CODE REVIEW

**Comment by skopacz1** [comment]
@gpei could you PTAL? Thanks!

cc @shellyyang1989

File: modules/rhel-compute-updating.adoc [hunk 1]
```diff
@@ -52,7 +52,7 @@ By default, the base OS RHEL with "Minimal" installation option enables firewall
 +
 [source,terminal,subs="attributes+"]
 ----
-# subscription-manager repos --disable=rhocp-4.13-for-rhel-8-x86_64-rpms \
+# subscription-manager repos --disable=rhocp-4.14-for-rhel-8-x86_64-rpms \
```
Comment (gpei) [inline modules/rhel-compute-updating.adoc:55]: The main branch should be for 4.16, right? If so, we need to update it as `4.15` here, and use `4.14` for 4.15 branch.

Comment (skopacz1) [reply modules/rhel-compute-updating.adoc:55]: Eventually when the 4.16 branch is created, it'll be based on main, that is true. However, until that branching happens, the main branch should only have changes made that apply to the latest existing branch, which is 4.15. 

So for this PR, the changes for 4.15 need to be made first in the main branch and then cherry-picked to 4.15. That's why I'm making the changes to `4.14` in these lines. Let me know if you have any questions about that, but this is mostly just the way the docs team manages its repository.

Comment (gpei) [reply modules/rhel-compute-updating.adoc:55]: Got it, it's ok for me if we're updating the docs in this way.

Comment (skopacz1) [reply modules/rhel-compute-updating.adoc:55]: Sounds good, thanks! Keeping the version numbers 4.14 in this PR for the 4.15+ doc change.

**Comment by michaelryanpeter** [comment]
LGTM

As a side note, it might be worth adding a {product-version-n1} to the `common-attributes` file. I know that I could use this as a variable in my docs.

/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed

**Pull request merged**